fearless/Modules/Staking/ControllerAccount/ControllerAccountPresenter.swift

```diff
@@ -139,6 +141,7 @@ extension ControllerAccountPresenter: ControllerAccountPresenterProtocol {
                 fee: fee,
                 locale: locale
             ),
+            dataValidatingFactory.controllerBalanceIsNotZero(controllerBalance, locale: locale),
```

Comment:
Le't also refresh in case of error like we do with fee, so we could handle cases when user adds tokens to the controller and then tries to proceed